cdk/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "cdk",
+  "version": "0.1.0",
```

Comment:
Please add a license element as explained in https://docs.npmjs.com/cli/v8/configuring-npm/package-json#license, for the MIT-0 license as in:

```json
{
  "license" : "MIT-0"
}
```

Reply:
Added in e9d4fff